streams/src/test/java/org/apache/kafka/streams/integration/GlobalThreadShutDownOrderTest.java

```diff
@@ -36,9 +36,8 @@
 import org.apache.kafka.streams.state.Stores;
 import org.apache.kafka.streams.state.internals.KeyValueStoreBuilder;
 import org.apache.kafka.test.IntegrationTest;
-import org.apache.kafka.test.MockProcessorSupplier;
+import org.apache.kafka.test.MockApiProcessorSupplier;
```

Comment:
I went ahead and just converted the mock processor to the new API. I gave it a temporary name. Eventually, the old one will be unused, and we can delete the old one and rename the new one back to `MockProcessorSupplier`.